packages/surveys/src/utils/status-utils.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ISurveyFormJson,
+  ISurveySchedule,
+  ISurveyStatus,
+  IDetailedSurveyStatus
+} from "@esri/hub-types";
```

Comment:
ok wasn't sure about where interfaces should go. @drewctate metioned `hub-types` as a landing spot for them. What's the rule of thumb for deciding where things go?

Reply:
So

Rule of thumb should be as adjacent as possible to the fns that take/return the types.

Specifically:

1. I'm going to move hub-types into hub-common, so definitely don't use that package
1. If the types are shared amongst pacakges, use hub-common
1. Beyond that they should go in the module w/ the fns that take/return them, b/c likely they are only needed by those fns and/or the consumers of that fns
1. In rare cases we may have multiple modules w/in a package that need the same interface/type but one of the modules should not import from another (i.e. it would cause a circular dependency), in that case put the type in a `src/types.ts` in that package

Reply:
immensely helpful, thanks @tomwayson